packages/cdk-assets/lib/private/handlers/files.ts

```diff
@@ -111,6 +135,39 @@ async function bucketOwnership(s3: AWS.S3, bucket: string): Promise<BucketOwners
   }
 }
 
+enum BucketEncryption {
+  NO_ENCRYPTION,
+  SSEAlgorithm_AES256,
+  SSEAlgorithm_aws_kms,
+  ACCES_DENIED,
+  DOES_NOT_EXIST
+}
+
+async function bucketEncryption(s3: AWS.S3, bucket: string) {
```

Comment:
Can we add caching to this function so we're not going to do the same lookup more than once?

Reply:
there are a lot of methods not being cached like bucketOwner etc. If we add caching it would make sense to add it to a level which ensures that the cache does have an effect. Right now I do not understand the system well enough to add caching.

Reply:
Fair enough. Will do that myself then.